### DISCUSSION TITLE
What's new in 2211.40000.10.0 ?

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/WSA#183] [General]
Just got the notification that my WSA was updated to version 2211.40000.10.0 but I can't find a change log for it.

I want to know what's changed in this version.

**Comment by nieubank** [comment]
There's a comment in the announcement of 2211. The primary changes here are updating to Android 13. 

https://github.com/microsoft/WSA/discussions/167

**Reply by ghost** [reply]
I was already on Android 13 since I had version 2211.40000.7.0 installed, but the update I received is 2211.40000.10.0

I can't find the comment you are referring to, all the comments mentioning build 2211.40000.10.0 is for its direct download links.

**Reply by nieubank** [reply]
This was a very minor update that addressed some reliability issues, specifically in regards to apps creating jump list entries could cause WSA to crash.

**Reply by ghost** [reply]
Thank you!
